src/components/Notification/Wrappers/PanelWrapper.js

```diff
@@ -34,6 +35,8 @@ const PanelWrapper = ({
     return '1 Unread Event';
   };
 
+  const getPanelHeight = () => accordionHeight - (panelCount * 55 + 41);
```

Comment:
The numbers 55 and 41 are very specific and could be different in different rendering contexts. Is it possible to capture this value when the component is rendered?

@jeff-phillips-18 is that something we would want to do in this case (i.e. get actual height of drawer header and panel header to use here)?

Reply:
Yeah, we can't assume the heights of the heading and footer. Each could be different based on contents.

Reply:
The other thing I'm noticing is that the panel header heights could vary depending on the length of the text string (e.g. if it wraps) and also the state of the drawer being normal width or wide width (e.g. a long string wraps at normal width but doesn't at wide width).

Reply:
Right,
I think i can keep going with `getElementsByClassName`, since it returns an array of the findings (each of them can have different height), i can get the exact height we need for the panel. ill update tonight